tests/functional/s3cmd/tests.js

```diff
@@ -775,3 +777,21 @@ describe('s3cmd recursive delete with objects put by MPU', () => {
         deleteFile(upload16MB, done);
     });
 });
+
+describeSkipIfOldConfig('If no location is sent with the request', () => {
+    beforeEach(done => {
+        exec(['mb', `s3://${bucket}`], done);
+    });
+    afterEach(done => {
+        exec(['rb', `s3://${bucket}`], done);
+    });
+    // WARNING: change "file" to another locationConstrain depending
+    // on the restEndpoints (./config.json)
+    it('endpoint should be used to determine the locationConstrain', done => {
+        checkRawOutput(['info', `s3://${bucket}`], 'Location', 'file',
```

Comment:
It's true that if we merge this PR first the s3cmd info tests may actually work, which might be nice. We would just have to wait a while to merge the other one.